extensions/metadata_exchange/build_wasm.sh

```diff
@@ -16,5 +16,11 @@
 
 set -e
 
-docker run -e uid="$(id -u)" -e gid="$(id -g)" -v $PWD:/work -w /work -v $(realpath $PWD/../../extensions):/work/extensions gcr.io/istio-testing/wasmsdk:v3 bash /build_wasm.sh
+# Get SHA of envoy-wasm repo and use that as wasm sdk image tag.
```

Comment:
Can we extract this part to a common util script?

Reply:
Done.